Src/SyntaxFinder/Program.cs

```diff
@@ -1,259 +1,32 @@
-﻿using System;
-using System.Collections.Concurrent;
-using System.Collections.Generic;
-using System.IO;
-using System.Linq;
-using System.Threading;
-using System.Threading.Tasks;
-using Microsoft.CodeAnalysis;
-using Microsoft.CodeAnalysis.CSharp;
-using Microsoft.CodeAnalysis.CSharp.Syntax;
-
-namespace SyntaxFinder;
+﻿using Microsoft.CodeAnalysis.CSharp;
+using SyntaxFinder;
 
 // Customize this in some way to find variations of syntax
 // csharpier-repos isn't a huge result set, but it can give you
 // some idea what syntax people prefer
 // make sure csharpier-repos is on main before running
-public class Program
-{
-    public static void Main()
-    {
-        var files = Directory.EnumerateFiles(
-            "C:\\projects\\csharpier-repos",
-            //@"C:\projects\insite-commerce-csharpier\Legacy\Core\Insite.Core.Interfaces\Data",
-            "*.cs",
-            SearchOption.AllDirectories
-        );
-
-        Parallel.ForEach(
-            files,
-            file =>
-            {
-                if (Ignored.Is(file))
-                {
-                    return;
-                }
-
-                var contents = File.ReadAllText(file);
-                var tree = CSharpSyntaxTree.ParseText(contents);
-
-                var customWalker = new CustomWalker(file);
-                customWalker.Visit(tree.GetRoot());
-            }
-        );
-
-        foreach (var entry in CustomWalker.MembersInType)
-        {
-            Console.WriteLine(entry.Key);
-            foreach (var member in entry.Value.OrderBy(o => o))
-            {
-                Console.WriteLine("    " + member);
-            }
-        }
-
-        if (CustomWalker.Matching > CustomWalker.Total)
-        {
-            Console.WriteLine("Matching was > than Total, so you did something wrong.");
-        }
-
-        WriteResult("Matching", CustomWalker.Matching.ToString("n0"));
-        WriteResult("Total", CustomWalker.Total.ToString("n0"));
-        WriteResult(
-            "Percent",
-            (Convert.ToDecimal(CustomWalker.Matching) / CustomWalker.Total * 100).ToString("n")
-                + "%"
-        );
-    }
 
-    private static void WriteResult(string label, string value)
-    {
-        Console.WriteLine((label + ":").PadRight(20) + value.PadLeft(10));
-    }
-}
+var files = Directory.EnumerateFiles(
+    "C:\\projects\\csharpier-repos",
```

Comment:
Uhm... could you add a readme file explaining that this is tool for finding test cases in csharpier-repos?
Maybe a sentence or two explaining how to use it (either here, or in the contributing/development doc) along with maybe a link to [the repo](https://github.com/belav/csharpier-repos)?

Reply:
Actually, you want to split that out into a separate PR? 
This will make no sense for anyone looking at the history at a later date.